base/dft.jl

```diff
@@ -395,8 +395,15 @@ if Base.USE_GPL_LIBS
 
     A multidimensional FFT simply performs this operation along each transformed dimension of `A`.
 
-    Higher performance is usually possible with multi-threading. Use `FFTW.set_num_threads(np)`
-    to use `np` threads, if you have `np` processors.
+    !!! note
+        * Julia starts FFTW up with 1 thread by default. Higher performance is usually possible by
+          increasing number of threads. Use `FFTW.set_num_threads(Sys.CPU_CORES)` to use as many
+          threads as cores on your system.
+
+        * This performs a multidimensional FFT by default. Other languages such as Python and Octave
```

Comment:
@tkelman Do you think it's necessary to specify that? I thought it was obvious in that when you say Python you're also talking about its ecosystem, right?